src/test/java/com/cloudbees/plugins/credentials/CredentialsProviderTest.java

```diff
@@ -285,7 +285,7 @@ public void testNodeCredentialFingerprintsAreRemovedForNonExistentNodes() throws
         DumbSlave addedSlave = new DumbSlave("added-slave",
                 "dummy", "/home/test/slave", "1", Node.Mode.NORMAL, "remote",
                 new JNLPLauncher(),
-                RetentionStrategy.INSTANCE, Collections.<NodeProperty<?>>emptyList());
+                RetentionStrategy.INSTANCE, Collections.emptyList());
```

Comment:
FYI, this constructor is deprecated and you can more easily use a simpler version.